### PR TITLE
Test coverage >=80%

### PR DESCRIPTION
**Pull request by kalachou**

### CODE REVIEW

File: src-fe/src/app/container/store/sample/sample.reducer.spec.ts [hunk 1]
```diff
@@ -7,6 +8,36 @@ describe('Sample Reducer', () => {
...
 
+			expect(result).toBe(initialState);
+		});
+	});
+	describe('LoadSamples action', () => {
+		it('should return the previous state', () => {
+			const action: any = {} as any;
+			action.type = SampleActionTypes.LoadSamples;
+
+			const result: any = reducer(initialState, action);
```
Comment (vladimir-sviridenko) [inline src-fe/src/app/container/store/sample/sample.reducer.spec.ts:19]: Get rid of 'any' types, if possible, please

File: src-fe/src/app/container/store/sample/sample.actions.spec.ts [hunk 2]
```diff
@@ -4,4 +4,10 @@ describe('Sample', () => {
 	it('should create an instance', () => {
 		expect(new SampleActions.LoadSamples()).toBeTruthy();
 	});
+	it('should create an instance', () => {
```
Comment (vladimir-sviridenko) [inline src-fe/src/app/container/store/sample/sample.actions.spec.ts:7]: There is three similar test description. Maybe better to write 'should create an instance of LoadSamplesFailure'.
The same below.

File: src-fe/src/app/container/store/sample/sample.effects.ts [hunk 3]
```diff
@@ -7,16 +7,16 @@ import { LoadSamplesFailure, LoadSamplesSuccess, SampleActionTypes, SampleAction
...
+// 	@Effect()
+// 	public loadSamples$: Observable<LoadSamplesSuccess | LoadSamplesFailure> = this.actions$.pipe(
+// 		ofType(SampleActionTypes.LoadSamples),
+// 		concatMap(() =>
+// 			/** An EMPTY observable only emits completion. Replace with your own observable API request */
+// 			EMPTY.pipe(
+// 				map((data: unknown) => new LoadSamplesSuccess({ data })),
+// 				catchError((error: Error) => of(new LoadSamplesFailure({ error }))))
+// 		)
+// 	);
```
Comment (vladimir-sviridenko) [inline src-fe/src/app/container/store/sample/sample.effects.ts:19]: Remove commented code

**Review by vladimir-sviridenko — CHANGES_REQUESTED** [review]

File: src-fe/src/app/container/store/sample/sample.reducer.spec.ts (hunk 1 above)
Comment (clain-atomi) [reply src-fe/src/app/container/store/sample/sample.reducer.spec.ts:19]: fixed?

File: src-fe/src/app/container/store/sample/sample.actions.spec.ts [hunk 4]
```diff
@@ -1,7 +1,13 @@
 import * as SampleActions from './sample.actions';
 
 describe('Sample', () => {
-	it('should create an instance', () => {
+	it('should create an instance LoadSamples', () => {
 		expect(new SampleActions.LoadSamples()).toBeTruthy();
 	});
+	it('should create an instance LoadSamplesFailure', () => {
```
Comment (clain-atomi) [inline src-fe/src/app/container/store/sample/sample.actions.spec.ts:7]: add new line between `it` and `describe` statements

**Review by clain-atomi — CHANGES_REQUESTED** [review]

**Review by clain-atomi — APPROVED** [review]

**Review by vladimir-sviridenko — APPROVED** [review]

**Pull request merged**